salt/__init__.py

```diff
@@ -3,9 +3,13 @@
 """
 
 import importlib
+import logging
+import os
 import sys
 import warnings
 
+log = logging.getLogger()
+
```

Comment:
```suggestion
```

Reply:
This is still valid, remove logging.

Reply:
Missed left over from debugging, it is gone